### ISSUE TITLE
Google Deprecating Place Picker

### ISSUE THREAD

**Issue by zhigang1992** [q6112345/react-native-google-place-picker#41]
https://developers.google.com/places/ios-sdk/client-migration#update_to_the_latest_places_client_library

**Comment by zhigang1992** [comment]
https://github.com/zhangtaii/react-native-google-place-picker/issues/21

This might be also related because newly created api will no longer work with location picker

**Comment by zhigang1992** [comment]
@zhangtaii imo, we should put a baner on the readme.md. just wasted a whole morning trying to figure this out. 🤦‍♂️

**Comment by akiladevinda** [comment]
In my case I still have this issue. @zhigang1992 is you know how to fix this issue ?

**Comment by zhigang1992** [comment]
@akiladevinda, unfortunately, there is no fix on this issue. You'd have to build your own picker.

**Comment by GaspardC** [comment]
@zhigang1992 so the current picker will not work after July 29, 2019? is this correct?

**Comment by zhigang1992** [comment]
@GaspardC yes and also it's newly applied API won't be able to access the picker at all
